Plots/Overlays/NCL_overlay_6.py

```diff
@@ -131,19 +130,19 @@
                            add_labels=False,
                            zorder=1)
 plt.colorbar(pressure, cax=cax1, ticks=np.arange(980, 1045, 5))
+
 # Format color bar label
 cax1.yaxis.set_label_text(label='\n'.join('Sea Level Pressure'),
                           fontsize=14,
                           rotation=0)
 cax1.yaxis.set_label_coords(-0.5, 0.9)
 cax1.tick_params(size=0)
 
-#
+###############################################################################
 # Overlay streamlines
-#
+
 with np.errstate(
-        invalid='ignore'
-):  # Indeed not needed, just to get rid of warnings about numpy's NaN comparisons
```

Comment:
Why did you remove this comment? It is necessary to explain to novice programmers why we are choosing to ignore warnings. This PR is just to fix issues with the `extend` and `norm` arguments for colorbars, so let's fix other things in a separate PR.